code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -416,3 +420,105 @@
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+		if("blue")
+			new /obj/item/clothing/head/wizard(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe(get_turf(src))
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+		if("red")
+			new /obj/item/clothing/head/wizard/red(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe/red(get_turf(src))
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+		if("necromancer")
+			new /obj/item/clothing/head/wizard/necro(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe/necro(get_turf(src))
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+		if("clown")
+			new /obj/item/clothing/head/wizard/clown(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe/clown(get_turf(src))
+			new /obj/item/clothing/mask/gas/clown_hat/wiz(get_turf(src))
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+		if("purple")
+			new /obj/item/clothing/head/wizard/amp(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe/psypurple(get_turf(src))
+			new /obj/item/clothing/shoes/sandal/marisa/leather(get_turf(src))
+		if("lich")
+			new /obj/item/clothing/head/wizard/lich(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe/lich(get_turf(src))
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+		if("skelelich")
+			new /obj/item/clothing/head/wizard/skelelich(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe/skelelich(get_turf(src))
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+		if("marisa")
+			new /obj/item/clothing/head/wizard/marisa(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe/marisa(get_turf(src))
+			new /obj/item/clothing/shoes/sandal/marisa(get_turf(src))
+		if("fake")
+			new /obj/item/clothing/head/wizard/fake(get_turf(src))
+			new /obj/item/clothing/suit/wizrobe/fake(get_turf(src))
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+
+	switch (randomizeartifact)
+//		if("staffchange")
+//			new /obj/item/weapon/gun/energy/staff/change(get_turf(src))
+		if("staffswap")
+			new /obj/item/weapon/gun/energy/staff/swapper(get_turf(src))
+		if("staffmental")
+			new /obj/item/weapon/gun/energy/staff/focus(get_turf(src))
+		if("soulstone")
+			new /obj/item/weapon/storage/belt/soulstone/full(get_turf(src))
+			add_spell(new /spell/aoe_turf/conjure/construct, iswizard = TRUE)
+			add_language(LANGUAGE_CULT)
+		if("rigsuit")
+			new /obj/item/clothing/shoes/sandal(get_turf(src))
+			new /obj/item/clothing/gloves/purple/wizard(get_turf(src))
+			new /obj/item/clothing/suit/space/rig/wizard(get_turf(src))
+			new /obj/item/weapon/tank/emergency_oxygen/double/wizard(get_turf(src))
+		if("staffanimate")
+			new /obj/item/weapon/gun/energy/staff/animate(get_turf(src))
+		if("staffnecro")
+			new /obj/item/weapon/gun/energy/staff/necro(get_turf(src))
+		if("apprentice") //lol
+			new /obj/item/wizard_apprentice_contract(get_turf(src))
+		if("scyingorb")
+			new /obj/item/weapon/scrying(get_turf(src))
+			mutations.Add(M_XRAY)
```

Comment:
this is the effect that holding the scrying orb gives you, why is this here?

Reply:
the scying orb doesn't do anything when held, it lets you ghost when you use it. It gives a wizard x-ray vision when they purchase it, so I'm giving the same perk to people who recieve it

Reply:
Huh, I remembered wrong